examples/README.md

```diff
@@ -49,7 +49,7 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-gke.yaml](#hpc-gkeyaml--) ![community-badge] ![experimental-badge]
   * [ml-gke](#ml-gkeyaml--) ![community-badge] ![experimental-badge]
   * [storage-gke](#storage-gkeyaml--) ![community-badge] ![experimental-badge]
-  * [htc-slurm.yaml](#htc-slurmyaml--) ![community-badge] ![experimental-badge]
+  * [htc-slurm-v5-legacy.yaml](#htc-slurm-v5-legacyyaml--) ![community-badge] ![deprecated-badge]
```

Comment:
This does not align with what we said: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/examples

> Three months after Slurm-gcp V6 becomes the recommended version, Slurm v5 modules will be marked as deprecated and will be maintained in our repo for another three months, at which point the modules will be removed from the HPC Toolkit repo and regression tests will no longer run for V5.

I read this as we are going to support for the next 3 months, I would not mark as deprecated yet.

Reply:
Removed deprecated badge.